README.md

```diff
@@ -1,3 +1,5 @@
 # test-01
 
 jhhjkjhjkhkjs
+
+;wjkjewhkjeh
```

Comment:
The added lines seem to be random characters. Could you please clarify their purpose and provide more context on the changes made?